src/Sentry.Tunnel/Sentry.Tunnel.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.0;netstandard2.0</TargetFrameworks>
+    <PackageTags>$(PackageTags);AspNetCore;MVC</PackageTags>
+    <PackageId>Sentry.Tunnel</PackageId>
+    <AssemblyName>Sentry.Tunnel</AssemblyName>
+    <RootNamespace>Sentry.Tunnel</RootNamespace>
```

Comment:
I think we don't need these anymore (likely can remove from all packages, not 100% sure)

```suggestion
```

Reply:
Since all the other projects use it, I'll keep it.